### ISSUE TITLE
Support: Public Docker images

### ISSUE THREAD

**Issue by pataquets** [fullstorydev/emulators#22]
I'd like to run the Object Storage emaulator.
Is there any pullable public Docker image available or do I have to build them locally?

If there isn't any available, would be such feature considered for inclusion?
It is feasible with some Github Actions magic and a Docker Hub (free plan) account, which could build & push them automatically on each commit.

**Comment by dragonsinth** [comment]
Yeah we've done similar things in https://github.com/fullstorydev/grpcurl so it seems feasible in theory.  Dunno if there's enough demand to be worth the lift.

**Comment by aarontatar-upguard** [comment]
It would be great to have this as a public docker image. It looks like there already is one, it's just quite old: https://registry.hub.docker.com/r/fullstorydev/cbtemulator. It'd be very nice to have an arm64 build, too. 

Someone's done this for the gcloud version of the emulator, but it doesn't have persistence. Maybe you can copy their GitHub actions: 
https://github.com/ridedott/bigtable-emulator-docker
https://github.com/ridedott/bigtable-emulator-docker/pkgs/container/cloud-bigtable-emulator 

(I'm happy to look into it and send a PR, too)

**Comment by dragonsinth** [comment]
Oh that's funny, I had no idea.  Looks like @franklinlindemberg pushed it.  Huh, we'll have to check it out.. it looks like there are Docker build rules.  @gpassini do you know how to actually push to docker hub?

**Comment by franklinlindemberg** [comment]
@dragonsinth @gpassini This is the original [PR ](https://github.com/fullstorydev/emulators/pull/1)where I added the dockerfiles,

This one is when I added the release script https://github.com/fullstorydev/emulators/pull/2

I don't recall the details, but I think we had some issues with this strategy in this project. in any case, you might need to add it again since it seems to have been removed

**Comment by dragonsinth** [comment]
Hi Franklin!! 😄 👋

**Comment by franklinlindemberg** [comment]
@dragonsinth Hey man! let me know if you need any help on this, happy to help!
